Frontend/src/pages/marketplace/ProductDetail.jsx

```
@@ -238,25 +238,27 @@ const ProductDetail = () => {
     { rating: 1, value: 2, percentage: "2%" },
   ];
+
   return (
     <div className="container mx-auto px-4 py-8 mt-10">
       {/* Breadcrumb */}
       <div className="flex items-center gap-1 text-sm mb-4">
         <a href="/" className="hover:underline">
           Shop all
         </a>
-        <span>&gt;</span>
+        <span></span>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)